.travis.yml

```diff
@@ -56,6 +56,8 @@ after_success:
 
 before_deploy:
   # TODO: cross build
+ - export CFG_RELEASE_CHANNEL=nightly
```

Comment:
Should this be up above in the `script` section?

Reply:
Yeah good catch, thanks!